### PR TITLE
[pylint] Convert remaining content to f-strings (part 3)

### PR DESCRIPTION
**Pull request by arif-ali**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [ ] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?
- [ ] Are all passwords or private data gathered by this PR [obfuscated](https://github.com/sosreport/sos/wiki/How-to-Write-a-Plugin#how-to-prevent-collecting-passwords)?

### CODE REVIEW

File: sos/policies/runtimes/__init__.py [hunk 1]
```diff
@@ -245,8 +245,7 @@ def get_copy_command(self, container, path, dest, sizelimit=None):
         :rtype:     ``str``
         """
         if sizelimit:
-            return "%s %s tail -c %s %s" % (self.run_cmd, container, sizelimit,
-                                            path)
-        return "%s cp %s:%s %s" % (self.binary, container, path, dest)
+            return f"{self.run_cmd} {container} tail -c {sizelimit} {path}"
+        return "{self.binary} cp {container}:{path} {dest}"
```
Comment (pmoravec) [inline sos/policies/runtimes/__init__.py:249]: Missing leading `f`.

**Comment by arif-ali** [comment]
good catches, should be resolved now

File: sos/report/__init__.py [hunk 2]
```diff
@@ -1081,8 +1077,8 @@ def list_plugins(self):
                 if tmpopt is None:
                     tmpopt = 0
 
-                self.ui_log.info(" %-25s %-15s %s" % (
-                    opt.plugin + "." + opt.name, tmpopt, opt.desc))
+                self.ui_log.info(f" {opt.plugin}.{opt.name:<25} {tmpopt:<15} "
```
Comment (pmoravec) [inline sos/report/__init__.py:1080]: Currently, we apply the "right alignment to 25characters" rule to `{opt.plugin}.{opt.name}`. Newly we will apply it just to `{opt.name}`.

We might have there

    f" {f'{opt.plugin}.{opt.name}':<25} {tmpopt:<15}"

or use a new variable.

File: sos/report/__init__.py [hunk 3]
```diff
@@ -1362,11 +1357,8 @@ def collect_plugin(self, plugin):
         except Exception:
             return False
         numplugs = len(self.loaded_plugins)
-        status_line = "  Starting %-5s %-15s %s" % (
-            "%d/%d" % (count, numplugs),
-            plugname,
-            "[Running: %s]" % ' '.join(p for p in self.running_plugs)
-        )
+        status_line = (f"  Starting {count}/{numplugs:<5} {plugname:<15} "
```
Comment (pmoravec) [inline sos/report/__init__.py:1360]: Same like above. Now we limit the size of whole `{count}/{numplugs}`, newly we would limit just `{numplugs}`.

**Review by pmoravec — COMMENTED** [review]
ACK up to the two latest comments. Great ant work!

**Review by pmoravec — APPROVED** [review]

**Review by TurboTurtle — APPROVED** [review]

**Comment by arif-ali** [comment]
rebasing due to conflict, and centos-stream-8 based jobs keeps timing out, 2nd restart

**Pull request merged**